models/prostate_mri_anatomy/scripts/center_crop.py

```diff
@@ -0,0 +1,71 @@
+import argparse
+from typing import Union
+
+import SimpleITK as sitk  # noqa N813
```

Comment:
I think we should add it to the optional packages of `metadata.json` even it's not used in the config files.
@wyli @ericspod What do you think?

Thanks.

Reply:
the cropping step is optional and I think it's ok to not include it in the metadata.

Reply:
OK, sounds good to me, I didn't notify it's optional.